pyfolio/pos.py

```diff
@@ -57,13 +57,14 @@ def get_long_short_pos(positions):
 
     pos_wo_cash = positions.drop('cash', axis=1)
     longs = pos_wo_cash[pos_wo_cash > 0].sum(axis=1).fillna(0)
-    shorts = pos_wo_cash[pos_wo_cash < 0].abs().sum(axis=1).fillna(0)
+    shorts = pos_wo_cash[pos_wo_cash < 0].sum(axis=1).fillna(0)
```

Comment:
Yes, it will. One of the issue tasks is to make the "shorts" line show up
as negative instead of positive.

"Make Long exposure line positive number, and Short exposure line all
negative numbers"

On Mon, May 23, 2016 at 3:34 AM, Thomas Wiecki <notifications@github.com>
wrote:

> In pyfolio/pos.py
> <https://github.com/quantopian/pyfolio/pull/301#discussion_r64178908>:
>
> > @@ -57,13 +57,14 @@ def get_long_short_pos(positions):
> >
> >      pos_wo_cash = positions.drop('cash', axis=1)
> >      longs = pos_wo_cash[pos_wo_cash > 0].sum(axis=1).fillna(0)
> > -    shorts = pos_wo_cash[pos_wo_cash < 0].abs().sum(axis=1).fillna(0)
> > +    shorts = pos_wo_cash[pos_wo_cash < 0].sum(axis=1).fillna(0)
>
> Won't removing abs() not change the results of the plot?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/quantopian/pyfolio/pull/301/files/12907415ecd871a19bd60e39c35613cc13692168#r64178908>
>